src/Schema/Schema.sqlproj

```diff
@@ -62,13 +62,17 @@
     <Folder Include="Triggers" />
     <Folder Include="Views" />
     <Folder Include="SchemaConversion" />
+    <Folder Include="Tables\Bills" />
   </ItemGroup>
   <ItemGroup>
     <None Include="SchemaConversion\ClientSchemaConversion.sql" />
   </ItemGroup>
   <ItemGroup>
     <Build Include="Tables\UsersLogin.sql" />
     <Build Include="Tables\Roles.sql" />
+    <Build Include="Tables\Bills\PatientBills.sql" />

```

Comment:
@Anastasiia37 @Kropisha @smoukiDev @FSuhov @ArteIIo @DenyaAbramenkov @MaksimTsedov
Commonly it makes sense to create a specific folder when you have at least 3-5 logic united tables.